x-pack/plugin/ml/qa/ml-with-security/build.gradle

```diff
@@ -17,7 +18,10 @@ task copyMlRestTests(type: Copy) {
   include 'rest-api-spec/test/ml/**'
 }
 
-integTestRunner {
+integTest {
+  dependsOn copyMlRestTests
```

Comment:
I don't think this is right. It should be the _runner_ that depends on this task, not the `RestIntegTestTask` which doesn't actually do anything. We should move this down inside the `runner { }` block.